wvdecrypter/cdm/media/cdm/cdm_adapter.cc

```diff
@@ -128,6 +137,8 @@ CdmAdapter::CdmAdapter(
 
 CdmAdapter::~CdmAdapter()
 {
+  exit_thread_flag = true;
+  std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
This value appears arbitrary. What exactly is this waiting for?

This value could very well work on one system but not on another. It would be better to continue once a condition is satisfied.

Reply:
I added another flag to check if timer thread is running or not. The sleep is/was for ensuring that the timer is not running anymore as otherwise cdm would get destroyed to early.